app/.gitignore

```diff
@@ -1 +0,0 @@
-/build
```

Comment:
Please don't delete this file too.

Reply:
This is also my mistake, same as for the service.